{"0.0.0":{"info":{"author":"LocalStack GmbH","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.0.0/","requires_dist":null,"requires_python":"","summary":"LocalStack extension that simulates region and service outages","version":"0.0.0","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"4a801669680a31110361f83f15185b8226676d723bd3c1f5f8174e984abec7ab","md5":"9d4e0e2df405481aa686a54f3180356a","sha256":"9a9ad29ac8a1c473ed3e1af4592bd495dd32765f4ca34d57f5bff72fb4127bc8"},"downloads":-1,"filename":"localstack-extension-outages-0.0.0.tar.gz","has_sig":false,"md5_digest":"9d4e0e2df405481aa686a54f3180356a","packagetype":"sdist","python_version":"source","requires_python":null,"size":3577,"upload_time":"2023-07-31T16:04:01","upload_time_iso_8601":"2023-07-31T16:04:01.079480Z","url":"https://files.pythonhosted.org/packages/4a/80/1669680a31110361f83f15185b8226676d723bd3c1f5f8174e984abec7ab/localstack-extension-outages-0.0.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.0":{"info":{"author":"LocalStack","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.1.0/","requires_dist":null,"requires_python":"","summary":"LocalStack extension that simulates region and service outages","version":"0.1.0","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"96392b8bb16f7f93b155275dab84f5ed23f58611f2747217ac3dfd83a299b72b","md5":"11713a6ea9426892660072266d49f381","sha256":"e1d048a9b39f8de16d0bc126bcd6880b5d99f42eb3cce6cd1e5791f7d3385e38"},"downloads":-1,"filename":"localstack-extension-outages-0.1.0.tar.gz","has_sig":false,"md5_digest":"11713a6ea9426892660072266d49f381","packagetype":"sdist","python_version":"source","requires_python":null,"size":2677,"upload_time":"2023-08-09T14:12:39","upload_time_iso_8601":"2023-08-09T14:12:39.820531Z","url":"https://files.pythonhosted.org/packages/96/39/2b8bb16f7f93b155275dab84f5ed23f58611f2747217ac3dfd83a299b72b/localstack-extension-outages-0.1.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.2.0":{"info":{"author":"LocalStack","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"text/markdown; charset=UTF-8","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.2.0/","requires_dist":null,"requires_python":"","summary":"LocalStack extension that simulates region and service outages","version":"0.2.0","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"2a7f3b770b3ab76b7099fe0d6e02a95eff1a479cf4eae74a5266dc05ed6979e8","md5":"9ccd91d8fe248255b7b12236f8e7c08c","sha256":"b293fd5e8a443723cefa61b87ccbcb96ff872771d6ee8d93b1df5600597bf0b1"},"downloads":-1,"filename":"localstack-extension-outages-0.2.0.tar.gz","has_sig":false,"md5_digest":"9ccd91d8fe248255b7b12236f8e7c08c","packagetype":"sdist","python_version":"source","requires_python":null,"size":2723,"upload_time":"2023-08-09T14:34:08","upload_time_iso_8601":"2023-08-09T14:34:08.563359Z","url":"https://files.pythonhosted.org/packages/2a/7f/3b770b3ab76b7099fe0d6e02a95eff1a479cf4eae74a5266dc05ed6979e8/localstack-extension-outages-0.2.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.2.1":{"info":{"author":"LocalStack","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"text/markdown","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.2.1/","requires_dist":null,"requires_python":"","summary":"LocalStack extension that simulates region and service outages","version":"0.2.1","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"179572bb483d6c54f60448a33df186c665078be02bfc7c7eeb862a2b36411cf9","md5":"88ea1e584c5450f50a3e686d05f8e262","sha256":"4dd1e3b505fa9cfc316564f2352d3c11f3bdabcbcba54fc6f4e7ca9eba89bdc7"},"downloads":-1,"filename":"localstack-extension-outages-0.2.1.tar.gz","has_sig":false,"md5_digest":"88ea1e584c5450f50a3e686d05f8e262","packagetype":"sdist","python_version":"source","requires_python":null,"size":2729,"upload_time":"2023-08-09T14:22:36","upload_time_iso_8601":"2023-08-09T14:22:36.984320Z","url":"https://files.pythonhosted.org/packages/17/95/72bb483d6c54f60448a33df186c665078be02bfc7c7eeb862a2b36411cf9/localstack-extension-outages-0.2.1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.3.0":{"info":{"author":"LocalStack","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"text/markdown; charset=UTF-8","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.3.0/","requires_dist":null,"requires_python":"","summary":"LocalStack extension that simulates region and service outages","version":"0.3.0","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"efa20e91d5cd3b09c3a8472cfcc62f724f90b7e8d4d000d076b3283fc1ea53f6","md5":"d654c5e767bcb3466c36c118517d3a64","sha256":"1ddfb55e004586a0903dc109b1366d18bb95f4e55aaa204d049f50c243349081"},"downloads":-1,"filename":"localstack-extension-outages-0.3.0.tar.gz","has_sig":false,"md5_digest":"d654c5e767bcb3466c36c118517d3a64","packagetype":"sdist","python_version":"source","requires_python":null,"size":3394,"upload_time":"2023-08-09T14:38:08","upload_time_iso_8601":"2023-08-09T14:38:08.071375Z","url":"https://files.pythonhosted.org/packages/ef/a2/0e91d5cd3b09c3a8472cfcc62f724f90b7e8d4d000d076b3283fc1ea53f6/localstack-extension-outages-0.3.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.4.0":{"info":{"author":"LocalStack","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"text/markdown; charset=UTF-8","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.4.0/","requires_dist":null,"requires_python":"","summary":"LocalStack Extension: Outages","version":"0.4.0","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"5c8740358ca4236435b86ab1a177d8cd41398ab000493e96820b0ed0477b5d57","md5":"fbbdb440a3ffdb5c79054f23d270278a","sha256":"332995549b4594ef1d6fa4bcdf0750191bac2144623f6a114c9876fbf095a6af"},"downloads":-1,"filename":"localstack-extension-outages-0.4.0.tar.gz","has_sig":false,"md5_digest":"fbbdb440a3ffdb5c79054f23d270278a","packagetype":"sdist","python_version":"source","requires_python":null,"size":3992,"upload_time":"2023-08-29T12:47:08","upload_time_iso_8601":"2023-08-29T12:47:08.566745Z","url":"https://files.pythonhosted.org/packages/5c/87/40358ca4236435b86ab1a177d8cd41398ab000493e96820b0ed0477b5d57/localstack-extension-outages-0.4.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.4.1":{"info":{"author":"LocalStack","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"text/markdown; charset=UTF-8","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.4.1/","requires_dist":null,"requires_python":"","summary":"LocalStack Extension: Outages","version":"0.4.1","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"7ad98d985d2ec2e9e049d4168ade4901e4f07388a3243023090af87d3b2f0ccb","md5":"8f48a46166a54b4e09f40a1975527623","sha256":"09740cc2f65df612f0a327026c375cfa5f14a5d574f34b95fe68517f910eeb1c"},"downloads":-1,"filename":"localstack-extension-outages-0.4.1.tar.gz","has_sig":false,"md5_digest":"8f48a46166a54b4e09f40a1975527623","packagetype":"sdist","python_version":"source","requires_python":null,"size":4070,"upload_time":"2023-11-16T13:14:47","upload_time_iso_8601":"2023-11-16T13:14:47.754210Z","url":"https://files.pythonhosted.org/packages/7a/d9/8d985d2ec2e9e049d4168ade4901e4f07388a3243023090af87d3b2f0ccb/localstack-extension-outages-0.4.1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.4.2":{"info":{"author":"LocalStack","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"text/markdown; charset=UTF-8","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.4.2/","requires_dist":["jsonschema>=4.18","black>=22.3.0; extra == \"dev\"","flake8>=6.0.0; extra == \"dev\"","isort>=5.12.0; extra == \"dev\"","localstack>=3.0.0.post2; extra == \"dev\"","localstack-core[runtime]>=3.0.0; extra == \"dev\"","localstack-ext>=3.0.0.post2; extra == \"dev\"","localstack-obfuscator>=0.2.0; extra == \"dev\"","pyproject-flake8>=6.0.0.post1; extra == \"dev\"","botocore>=1.31.2; extra == \"test\"","boto3>=1.26.121; extra == \"test\"","pytest~=6.2; extra == \"test\""],"requires_python":null,"summary":"LocalStack Extension: Outages","version":"0.4.2","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"4c6f515211f12ac4434a846998e61737014097fd398ea5982975f8611c89c7a2","md5":"2ee3f9daf3103d798dae27f49ce20558","sha256":"a11e1f705653b01bb192503bea95ca3da647b32ec9c3f69f82ce74f411575646"},"downloads":-1,"filename":"localstack_extension_outages-0.4.2.tar.gz","has_sig":false,"md5_digest":"2ee3f9daf3103d798dae27f49ce20558","packagetype":"sdist","python_version":"source","requires_python":null,"size":4181,"upload_time":"2024-07-23T10:13:25","upload_time_iso_8601":"2024-07-23T10:13:25.788571Z","url":"https://files.pythonhosted.org/packages/4c/6f/515211f12ac4434a846998e61737014097fd398ea5982975f8611c89c7a2/localstack_extension_outages-0.4.2.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.5.0":{"info":{"author":"LocalStack","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"text/markdown; charset=UTF-8","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.5.0/","requires_dist":null,"requires_python":null,"summary":"LocalStack Extension: Outages","version":"0.5.0","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"525ca3226f14dfaeebe739eda529617c77d90f8ec994131bc6919a393d2a3845","md5":"78df2490f9d0a1e7898b49937c9e81b1","sha256":"6a95389a7b5c501f7210f039a3d8abf433077bbba942e0d02ebae4c548536786"},"downloads":-1,"filename":"localstack-extension-outages-0.5.0.tar.gz","has_sig":false,"md5_digest":"78df2490f9d0a1e7898b49937c9e81b1","packagetype":"sdist","python_version":"source","requires_python":null,"size":4088,"upload_time":"2024-07-23T11:08:17","upload_time_iso_8601":"2024-07-23T11:08:17.192995Z","url":"https://files.pythonhosted.org/packages/52/5c/a3226f14dfaeebe739eda529617c77d90f8ec994131bc6919a393d2a3845/localstack-extension-outages-0.5.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.6.0":{"info":{"author":"LocalStack","author_email":"support@localstack.cloud","bugtrack_url":null,"classifiers":[],"description_content_type":"text/markdown; charset=UTF-8","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://localstack.cloud","keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"localstack-extension-outages","package_url":"https://pypi.org/project/localstack-extension-outages/","platform":null,"project_url":"https://pypi.org/project/localstack-extension-outages/","project_urls":{"Homepage":"https://localstack.cloud"},"provides_extra":null,"release_url":"https://pypi.org/project/localstack-extension-outages/0.6.0/","requires_dist":null,"requires_python":null,"summary":"LocalStack Extension: Outages","version":"0.6.0","yanked":false,"yanked_reason":null},"last_serial":24302927,"urls":[{"comment_text":"","digests":{"blake2b_256":"4fa6dd54c71f9c3d557d66bc57b37adcb313d38b2c2ac2f1bfc41bf524ca583c","md5":"48eb2c835de1367b99cd01e2cfcc1142","sha256":"f44c210630a9e5fd182f8ce1ed97f7c508ea555bca77bcf16f12d9e8918f2075"},"downloads":-1,"filename":"localstack-extension-outages-0.6.0.tar.gz","has_sig":false,"md5_digest":"48eb2c835de1367b99cd01e2cfcc1142","packagetype":"sdist","python_version":"source","requires_python":null,"size":4183,"upload_time":"2024-07-26T11:21:30","upload_time_iso_8601":"2024-07-26T11:21:30.089831Z","url":"https://files.pythonhosted.org/packages/4f/a6/dd54c71f9c3d557d66bc57b37adcb313d38b2c2ac2f1bfc41bf524ca583c/localstack-extension-outages-0.6.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}